tests/Unit/ControlSystem/Test_Metafunctions.cpp

```diff
@@ -52,5 +57,13 @@ static_assert(
 static_assert(std::is_same_v<
               control_systems_with_measurement_t<test_systems, MeasurementB>,
               tmpl::list<SystemB0>>);
+
+static_assert(
+    std::is_same_v<control_components<Metavariables, fewer_test_systems>,
+                   tmpl::list<ControlComponent<Metavariables, SystemA0>,
+                              ControlComponent<Metavariables, SystemB0>>> or
+    std::is_same_v<control_components<Metavariables, test_systems>,
+                   tmpl::list<ControlComponent<Metavariables, SystemB0>,
+                              ControlComponent<Metavariables, SystemA0>>>);
```

Comment:
Both branches here should be testing the same function call, yes?

Reply:
Oops yes I forgot to change the name!